doc/feature/pyCompartmentReport.md

```diff
@@ -0,0 +1,63 @@
+High-Level CompartmentReport API for Python {#compartment_report_py}
+============
+
+## Motivation
+
+Redesign the current BBPSDK-inspired API for also allowing voltage traces.
+The returned data should change to be a matrix[frame, compartment] instead
+of the current vector[compartments].
+
+## API
+
```

Comment:
To summarize the comment above:
* The report metadata is static, that doesn't work from stream reports. A possible solution is:
    * Adding getEndTime to the view could be the solution...
    * but it's not clear when to update the timestamp.
    * The static metadata would remain and endtime would be +inf for stream reports.
    * This solution contains some inconsistencies compared to SpikeReportReader. Maybe the same principles should be applied there.
* Functions to now when a report is ended and close a stream (to unlock any reader) are missing.